indico/modules/rb_new/client/js/modules/landing/reducers.js

```diff
@@ -38,4 +38,19 @@ export default combineReducers({
             }
         }
     }),
+    upcomingBookings: combineReducers({
+        request: requestReducer(
+            landingActions.FETCH_BOOKINGS_REQUEST,
+            landingActions.FETCH_BOOKINGS_SUCCESS,
+            landingActions.FETCH_BOOKINGS_ERROR
+        ),
+        data: (state = null, action) => {
```

Comment:
default to an empty list/object? (see my other comment on the component proptype)

Reply:
That's already the default in the `stats` key. And TBH, it's much easier to check for `!state` than for `!state.length`. So, I'd tend to prefer leaving it like this.